modules/creating-a-bias-metric-using-cli.adoc

```diff
@@ -72,7 +72,7 @@ curl -sk -H "Authorization: Bearer $TOKEN" -X POST --location $TRUSTY_ROUTE/metr
 `favorableOutcome`:: The value of the `outcomeName` output that describes the favorable or desired model prediction.
```

Comment:
Just wondering if it's necessary to use both "favorable" and "desired", instead of just "favorable"?